src/stravalib/tests/unit/test_model.py

```diff
@@ -29,10 +29,6 @@
 
 @pytest.mark.parametrize("model_class,attr,value", ((Club, "name", "foo"),))
 class TestLegacyModelSerialization:
-    def test_legacy_deserialize(self, model_class, attr, value):
```

Comment:
This test is now no longer needed

Reply:
i'm now not sure if this test should remain if we are indeed supporting this legacy behavior. i initially read the issue as we are not but perhaps we are.